drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/editor/strategies/AbstractDataManagementStrategyTest.java

```diff
@@ -90,6 +136,48 @@ public void getPropertiesToHideListPropertyAssigned() {
         commonGetPropertiesToHideList(true);
     }
 
+    @Test
+    public void storeData() {
+        ScenarioSimulationContext scenarioSimulationContextSpy = spy(scenarioSimulationContextLocal);
+        AbstractScesimGridModel abstractScesimGridModelMock = mock(AbstractScesimGridModel.class);
+        when(scenarioSimulationContextSpy.getSelectedScenarioGridModel()).thenReturn(abstractScesimGridModelMock);
+        final FactModelTuple factModelTuple = getFactTuple();
+        abstractDataManagementStrategySpy.storeData(factModelTuple, testToolsPresenterMock, scenarioSimulationContextSpy);
+        verify(testToolsPresenterMock, times(1)).setDataObjectFieldsMap(isA(SortedMap.class));
+        verify(testToolsPresenterMock, times(1)).setSimpleJavaTypeFieldsMap(isA(SortedMap.class));
+        verify(testToolsPresenterMock, times(1)).setInstanceFieldsMap(isA(SortedMap.class));
+        verify(testToolsPresenterMock, times(1)).setSimpleJavaInstanceFieldsMap(isA(SortedMap.class));
+        verify(testToolsPresenterMock, times(1)).setHiddenFieldsMap(isA(SortedMap.class));
+        verify(testToolsPresenterMock, times(1)).hideProperties(isA(Map.class));
+        verify(scenarioSimulationContextSpy, times(1)).setDataObjectFieldsMap(isA(SortedMap.class));
+        verify(scenarioSimulationContextSpy, times(1)).setDataObjectsInstancesName(isA(Set.class));
+        verify(abstractScesimGridModelMock, times(1)).setSimpleJavaTypeInstancesName(isA(Set.class));
+    }
+
+    @Test
+    public void getInstanceMap() {
+        SortedMap<String, FactModelTree> sourceMap = getSourceMap();
+        SortedMap<String, FactModelTree> retrieved = abstractDataManagementStrategySpy.getInstanceMap(sourceMap);
+        assertNotNull(retrieved);
+        assertTrue(retrieved.isEmpty());
+        abstractDataManagementStrategySpy.setModel(modelLocal);
+        retrieved = abstractDataManagementStrategySpy.getInstanceMap(sourceMap);
+        assertNotNull(retrieved);
+        // " is the number of factmappings - inside modelLocal - whose class is "Void"
+        assertEquals(2, retrieved.size());
+    }
+
+    private SortedMap<String, FactModelTree> getSourceMap() {
+        SortedMap<String, FactModelTree> toReturn = new TreeMap<>();
+        FactModelTree toPut = new FactModelTree("Void", "package", new HashMap<>(), new HashMap<>());
```

Comment:
could we please use `Collections.emptyMap()`

Reply:
@jomarko 
Incompatible type :  Collections  final <K,V> Map<K,V> emptyMap()